indexer-app/src/cmr/indexer/data/concepts/variable.clj

```diff
@@ -31,10 +31,19 @@
       (map #(merge base-doc (measurement-quantity->elastic-doc %)) quantities)
       [base-doc])))
 
+(defn- get-concept-id-revision-id-map
+ "Return the concept-id revision-id map."
+ [concept-id revision-id]
+ (util/remove-nil-keys
+  {:concept-id concept-id
+   :revision-id (if (string? revision-id)
+                  (read-string revision-id)
```

Comment:
why read-string, what is expected to be inside of revision-id that requires this?

Reply:
certain revision-ids are passed in as string.

Reply:
I understand now how read-string works.